### PR TITLE
feat: Implement 'NEAR' sign txn for transfer and add account fn call

### PR DESCRIPTION
**Pull request by kartikCypherock**
Implements: https://app.clickup.com/t/9002019994/PRF-4976
Requires: https://github.com/Cypherock/cypherock-common/pull/31

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-4976 Near Send Transaction](https://app.clickup.com/t/9002019994/PRF-4976)

**Comment by kartikCypherock** [comment]
Build issue is due to unmerged PR on the submodule: https://github.com/Cypherock/cypherock-common/pull/33

File: apps/near_app/near_txn.c [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+/*****************************************************************************
+ * STATIC FUNCTION PROTOTYPES
+ *****************************************************************************/
+
+/**
+ * @brief Checks if the provided query contains expected request.
+ * @details The function performs the check on the request type and if the check
+ * fails, then it will send an error to the host bitcoin app and return false.
+ *
+ * @param query Reference to an instance of btc_query_t containing query
```
Comment (ujjwal-cyph) [inline apps/near_app/near_txn.c:97]: `btc_query_t` left to rename

**Review by ujjwal-cyph — CHANGES_REQUESTED** [review]
please resolve conflicts

File: apps/near_app/near_txn.c (hunk 1 above)
Comment (kartikCypherock) [reply apps/near_app/near_txn.c]: updated

**Review by ujjwal-cyph — APPROVED** [review]

**Review by amanCypherock — APPROVED** [review]

**Pull request merged**